dist/src/main/java/io/camunda/zeebe/shared/security/SecurityConfiguration.java

```diff
@@ -117,16 +121,36 @@ public GatewaySecurityAuthenticationEnabledCondition() {
       super(ConfigurationPhase.REGISTER_BEAN);
     }
 
+    @Conditional(ProfileGatewayCondition.class)
     @ConditionalOnProperty(
         prefix = "zeebe.gateway",
         value = "security.authentication.mode",
-        havingValue = "none")
+        havingValue = "none",
+        matchIfMissing = true)
     static class StandaloneGatewayCondition {}
 
+    @Conditional(ProfileBrokerCondition.class)
```

Comment:
❓ Would this be the same `@Profile("broker")`, or is the implementation of `ProfileBrokerCondition` different?